plugins/glustershd/init.go

```diff
@@ -18,24 +19,23 @@ func (p *Plugin) Name() string {
 func (p *Plugin) RestRoutes() route.Routes {
 	return route.Routes{
 		route.Route{
-			Name:        "GlustershEnable",
-			Method:      "POST",
-			Pattern:     "/volumes/{name}/heal/enable",
-			Version:     1,
-			HandlerFunc: glustershEnableHandler},
+			Name:         "SelfHealInfo",
+			Method:       "GET",
+			Pattern:      "/volumes/{name}/{opts}/heal-info",
+			Version:      1,
+			ResponseType: utils.GetTypeString((*glustershdapi.Brick)(nil)),
```

Comment:
You have to re-generate API documentation.

Reply:
In both these cases, in the handler, what you are actually returning is a list/slice of `glustershd.Brick` type. Here it's a single instance. Please fix it later. I'll merge this as this incorrect value here doesn't affect functionality.